### ISSUE TITLE
[$30][MSFT-130] Registration- Screen reader is not reading the username tips special characters

### ISSUE THREAD

**Issue by veshu** [topcoder-platform/community-app#2827]
**Steps to Reproduce**
1. login to https://accounts.topcoder.com/member.
2. Click Join now.
3. Click on the username field.
4. Username tips dialog is shown.
5. Observe the screen reader by reading its text. Screen reader is not reading this line.
-_ {} []

**Expected Result**
Screen reader should read the last line with these charecters.
-_ {} []

**Actual Result**
Screen reader is not reading the last line having these charecters alone.
-_ {} []

**Device**: Desktop/Labtop
**Operating System**: Windows
**Browser**: Chrome

**WCAG Category**: 1. Perceivable
**WCAG Level**: AAA
**WCAG Success Criterion**: 1.2.7 Extended Audio Description (Prerecorded)
**Screen Reader Used**: Other
**Accessibility Audit Tool used**: N/A

**Attachment**
https://topcodermsft-my.sharepoint.com/personal/pd-topcoder_topcodermsft_onmicrosoft_com/Documents/Forms/All.aspx?cid=4c403434%2D55d3%2D4752%2Db379%2D635595c2fa3a&FolderCTID=0x0120005C598A51049FC14CBA882E1AEE168F51&id=%2Fpersonal%2Fpd%2Dtopcoder%5Ftopcodermsft%5Fonmicrosoft%5Fcom%2FDocuments%2FMSFT%2DTeams%2DQA%2FJune%202019%2F29%20Jun%2FTest%20Results%2FBug%20Videos%20%26%20Log%20Files%2Fjackgid16%2Daccessibility%2F5

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096268 has been created for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096268 has been updated - the new changes has been updated for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096268 has been updated - it has been assigned to sadathanwar17.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096268 has been updated - the new changes has been updated for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096268 has been updated - it has been assigned to afrisalyp.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[500]: Challenge for the updated issue 2827 is creating, rescheduling this event<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[400]: Failed to create challenge. Detail: Operation failed in the contest service facade.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[400]: Failed to create challenge. Detail: Operation failed in the contest service facade.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30097367 has been created for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30097367 has been updated - it has been assigned to afrisalyp.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by Oanh-and-only-Oanh** [comment]
please review code PR https://github.com/appirio-tech/accounts-app/pull/256

**Comment by nkumar-topcoder** [comment]
this has been pushed to prod, matt verified and this can be closed

**Comment by crazyk07** [comment]
Closed for payment

**Comment by crazyk07** [comment]
Payment task has been updated: https://software.topcoder.com/review/actions/ViewProjectDetails?pid=30097367<br/><br/>```This is an automated message for crazyk via Topcoder X```
